Added Ustecky kraj data 3 a 4

DarthProvader opened the pull request.

Commit:
Added Ustecky kraj

Mintaka commented on data_sources.py:
```diff
@@ -5,4 +5,8 @@
                              ],
+                 "Ustecky":[
+                            "U1SNEZ01_T_N.csv.zip",
+                            "U2VARN01_T_N.csv.zip"
```
Díky za commit.
Malá drobnost. Za "U2VARN01_T_N.csv.zip" bude dobré dát čárku. Ten, kdo by "Ustecky" kraj doplňoval o další zdroje, by musel sáhnout do Tvé řádky a tu čárku tam přidat. Je přehlednější, aby do ní nesahal, když to není nutné a pokud by si toho nevšiml a nepřidal ji, tak by nám vznikla chyba.

Oprava je snadná:
* přidat čárku ve tvém kódu
* udělat další commit do stejné větve
* udělat push

DarthProvader committed:
Added space

Tried adding space to see it it fixes issue with the comma. Should have been in first version already.

DarthProvader committed:
Removed space, added comma

Finally understand what is wanted from me. should be ok now...

DarthProvader commented:
Ahoj díky za upozornění. 
Až ten druhý commit by měl být správně, v prvním 0ffe24e jsem nějak nepochopil, co se po mně chce...

Mintaka reviewed (approved):
Díky. Vypadá to OK. Přidám to do main.
